src/components/QueueCard.js

```diff
@@ -45,7 +45,7 @@ const QueueCard = ({
       {...rest}
     >
       <CardBody>
-        <CardTitle className="d-flex flex-wrap align-items-center">
+        <CardTitle tag="h5" className="d-flex flex-wrap align-items-center">
```

Comment:
Reading the rationale in https://github.com/reactstrap/reactstrap/issues/1297, I think it might make most sense to use the Bootstrap `h5` class here, as these aren't really semantically titles on a page.